src/lib/Scenes/Artwork/Components/__tests__/OtherWorks-tests.tsx

```diff
@@ -0,0 +1,56 @@
+import { shallow } from "enzyme"
+import { ArtworkFixture } from "lib/__fixtures__/ArtworkFixture"
+import React from "react"
+import { ArtworkContextArtistFragmentContainer as ArtworkContextArtist } from "../OtherWorks/ArtworkContexts/ArtworkContextArtist"
+import { OtherWorksFragmentContainer as OtherWorks } from "../OtherWorks/index"
+
+describe("OtherWorks", () => {
+  it("renders ArtistArtworkGrid with correct components", () => {
+    const regularArtwork = {
+      ...ArtworkFixture,
+      context: null,
```

Comment:
would be good to add another success test case for `ArtworkContextArtist`

Reply:
i.e. with

```
context: {
   __typename: "ArtworkContextArtist",
},
```

rather than

```
context: null,
```

Reply:
ended up adding a separate test for that context, i saw `null` context in real data so going to keep that test around too 🙌